packages/material-react-table/src/inputs/MRT_FilterTextField.tsx

```diff
@@ -352,7 +365,26 @@ export const MRT_FilterTextField = <TData extends MRT_RowData>({
 
   return (
     <>
-      {isDateFilter ? (
+      {filterVariant?.startsWith('datetime') ? (
```

Comment:
some of these props being passed are identical to the datepicker one. With every byte of bundle size mattering a bit. We might look to make a `commonDateProps` object.